apps/bam.iobio/README.md

```diff
@@ -1,10 +1,40 @@
-# Overture Wrapper for Bam.Iobio Components
+# Overture Demo App for Bam.Iobio Components
 
-## Setup
+## Setup + Local Development
 
 - Install dependencies: `pnpm ci`
```

Comment:
I don't think pnpm has a `ci` command

Reply:
Good catch and from what I read this kind of flag is not really needed with pnpm